var/spack/repos/builtin/packages/sw4lite/package.py

```diff
@@ -23,32 +23,76 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
 from spack import *
+import glob
 
 
 class Sw4lite(MakefilePackage):
     """Sw4lite is a bare bone version of SW4 intended for testing
     performance optimizations in a few important numerical kernels of SW4."""
 
-    tags = ['proxy-app']
+    tags = ['proxy-app', 'ecp-proxy-app']
 
-    homepage = "https://github.com/geodynamics/sw4lite"
+    homepage = "https://geodynamics.org/cig/software/sw4"
     url      = "https://github.com/geodynamics/sw4lite/archive/v1.0.zip"
 
     version('develop', git='https://github.com/geodynamics/sw4lite',
             branch='master')
     version('1.0', '3d911165f4f2ff6d5f9c1bd56ab6723f')
 
+    variant('openmp', default=True, description='Build with OpenMP support')
+    variant('precision',
+        default='double',
+        values=('float', 'double'),
+        multi=False,
+        description='Floating point precision')
+    variant('ckernel',
+        default='no',
+        values=('yes', 'no'),
+        multi=False,
+        description='C or Fortran kernel')
```

Comment:
Just use the default boolean variant syntax, with default `False` and no values specified.